e2e/automated.test.js

```diff
@@ -44,6 +49,17 @@ describe('End to End tests', () => {
 
   describe('single blob', () => {
     fixtures.forEach(({ url, content, lineNumber, targetUrl }) => {
+      if (targetUrl === false) {
+        it(`does not resolves ${content}`, async () => {
```

Comment:
```suggestion
        it(`does not resolve ${content}`, async () => {
```
Maybe adjust the wording?